lib/plugin.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * @fileoverview set of 6river eslint plugins
+ * @author Nick Chistyakov
+ */
+'use strict';
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const requireIndex = require('requireindex');
+const fs = require('node:fs');
+const pkg = JSON.parse(fs.readFileSync(new URL('./package.json', import.meta.url), 'utf8'));
+
+//------------------------------------------------------------------------------
+// Plugin Definition
+//------------------------------------------------------------------------------
+
+const plugin = {
+	rules: requireIndex(__dirname + '/rules'),
+	meta: {
+		name: pkg.name,
+		// version: pkg.version,
```

Comment:
Any reason not to uncomment this?

Reply:
Gotta get CI to update the package version, unless that happens in run-release?

Reply:
Yes, run-release will patch up package.json:
https://github.com/6RiverSystems/circleci-orbs/blob/b58df14fba8c988c9e18727e1305bd21cfb8e916/src/scripts/run-release.sh#L42-L43

Reply:
dope